adafruit_fona/adafruit_fona.py

```diff
@@ -370,7 +374,7 @@ def gps(self):
 
     @gps.setter
     def gps(self, gps_on=False):
-        """Sets GPS module power, parses returned buffer.
+        """Enables or disables GPS module.
         :param bool gps_on: Enables the GPS module, disabled by default.
```

Comment:
A param here doesn't make sense because it's a setter. The default argument is unneeded because it's always provided.

Reply:
Also, I think this doc string is ignored in favor of the getter's doc string.

Reply:
@tannewt  Would this be the case with all setters?

Reply:
Yes

Reply:
Ok, thanks. Did a pass removing docstrings + parameters for setters.